src/MassTransit.TestFramework/MassTransit.TestFramework.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="../../signing.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net6.0;net8.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
```

Comment:
@phatboyg that might be something you won't like. But NUnit4 doesn't support `nestandard2.0` and this project is base for all other tests